### PR TITLE
Be more specific when toasting "Success!"

### PR DESCRIPTION
**Pull request by McLoo**

### CODE REVIEW

**Comment by moxie0** [comment]
needs rebase

**Comment by McLoo** [comment]
rebased

File: res/values/strings.xml [hunk 1]
```diff
@@ -259,7 +259,7 @@
     <string name="ExportFragment_exporting_plaintext_to_storage">Exporting plaintext to storage...</string>
     <string name="ExportFragment_error_unable_to_write_to_storage">Error, unable to write to storage.</string>
     <string name="ExportFragment_error_while_writing_to_storage">Error while writing to storage.</string>
-    <string name="ExportFragment_success">Success!</string>
+    <string name="ExportFragment_export_file_created_successfully">Export file created successfully.</string>
```
Comment (moxie0) [inline res/values/strings.xml]: should probably be "Export successful"

**Comment by McLoo** [comment]
done and rebased

**Pull request closed without merging**